### ISSUE TITLE
-version does not work.

### ISSUE THREAD

**Issue by jfy133** [Integrative-Transcriptomics/DamageProfiler#48]
I just tried adding 1.1 to bioconda, but the build is failing because during tests DamageProfiler is always reporting exit code 1 when I'm requesting `-h` or `--help`.

The Bioconda devs say this sometimes happens, and it's an OK build if e.g. you can grep for version (suggesting the tool is indeed executable) then we can proceed, however when I run `-version` I just get teh help message too and the error.  

```
james@spinosaurus:~/Downloads$ java -jar DamageProfiler-1.1-java11.jar -help
usage: User option parser
 -h                                   Shows this help page.
 -version                             Shows the version of DamageProfiler.
 -i <INPUT>                           The input sam/bam/cram file.
 -r <REFERENCE>                       The reference file (fasta format).
 -o <OUTPUT>                          The output folder.
 -t <THRESHOLD>                       DamagePlot: Number of bases which are considered for plotting nucleotide misincorporations.
                                      Default: 25
 -s <SPECIES>                         Reference sequence name (Reference NAME flag of SAM record). For more details see
                                      Documentation.
 -sf <SPECIES FILE>                   List of species reference names (Reference NAME flag of SAM record). For more details see
                                      Documentation.
 -l <LENGTH>                          Number of bases which are considered for frequency computations. Default: 100.
 -title <TITLE>                       Title used for all plots. Default: input filename.
 -yaxis_dp_max <MAX_VALUE>            DamagePlot: Maximal y-axis value.
 -color_c_t <COLOR_C_T>               DamagePlot: Color (HEX code) for C to T misincoporation frequency.
 -color_g_a <COLOR_G_A>               DamagePlot: Color (HEX code) for G to A misincoporation frequency.
 -color_insertions <COLOR_C_T>        DamagePlot: Color (HEX code) for base insertions.
 -color_deletions <COLOR_DELETIONS>   DamagePlot: Color (HEX code) for base deletions.
 -color_other <COLOR_OTHER>           DamagePlot: Color (HEX code) for other bases different to reference.
 -only_merged                         Use only mapped and merged (in case of paired-end sequencing) reads to calculate the damage
                                      plot instead of using all mapped reads. The SAM/BAM entry must start with 'M_', otherwise it
                                      will be skipped. Default: false
 -sslib                               Single-stranded library protocol was used. Default: false
Unrecognized option: -help
james@spinosaurus:~/Downloads$ java -jar DamageProfiler-1.1-java11.jar -version
usage: User option parser
 -h                                   Shows this help page.
 -version                             Shows the version of DamageProfiler.
 -i <INPUT>                           The input sam/bam/cram file.
 -r <REFERENCE>                       The reference file (fasta format).
 -o <OUTPUT>                          The output folder.
 -t <THRESHOLD>                       DamagePlot: Number of bases which are considered for plotting nucleotide misincorporations.
                                      Default: 25
 -s <SPECIES>                         Reference sequence name (Reference NAME flag of SAM record). For more details see
                                      Documentation.
 -sf <SPECIES FILE>                   List of species reference names (Reference NAME flag of SAM record). For more details see
                                      Documentation.
 -l <LENGTH>                          Number of bases which are considered for frequency computations. Default: 100.
 -title <TITLE>                       Title used for all plots. Default: input filename.
 -yaxis_dp_max <MAX_VALUE>            DamagePlot: Maximal y-axis value.
 -color_c_t <COLOR_C_T>               DamagePlot: Color (HEX code) for C to T misincoporation frequency.
 -color_g_a <COLOR_G_A>               DamagePlot: Color (HEX code) for G to A misincoporation frequency.
 -color_insertions <COLOR_C_T>        DamagePlot: Color (HEX code) for base insertions.
 -color_deletions <COLOR_DELETIONS>   DamagePlot: Color (HEX code) for base deletions.
 -color_other <COLOR_OTHER>           DamagePlot: Color (HEX code) for other bases different to reference.
 -only_merged                         Use only mapped and merged (in case of paired-end sequencing) reads to calculate the damage
                                      plot instead of using all mapped reads. The SAM/BAM entry must start with 'M_', otherwise it
                                      will be skipped. Default: false
 -sslib                               Single-stranded library protocol was used. Default: false
Missing required options: i, o
```
It would be good a) for the`-version` to actually be reported, and b) preferably running with `-h` gives a exit code 0 (as this is what should happen given that's what the user asked for).

**Comment by JudithNeukamm** [comment]
Argh....sorry. It's fixed now and the released version 1.1 is updated. Hope it's working now.

**Comment by jfy133** [comment]
Confirmed!
